src/main/kotlin/com/timcastelijns/room15bot/bot/usecases/MaukerUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.timcastelijns.room15bot.bot.usecases
+
+class MaukerUseCase: UseCase<String?, String> {
```

Comment:
If the usecase takes no input, use Unit instead of a nullable type

Reply:
True. Will do.